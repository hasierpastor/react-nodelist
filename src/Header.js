import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

const headerStyle = {
  fontFamily: 'raleway',
  fontSize: '4rem',
  margin: '1rem'
};

const iconStyle = {
  color: '#228B22'
};

//Simple reusable header component
function Header(props) {
  return (
    <h1 style={headerStyle}>
      Tree Table <FontAwesomeIcon icon="tree" style={iconStyle} />
    </h1>
  );
}

export default Header;
