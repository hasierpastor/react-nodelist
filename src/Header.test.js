import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './Header';

it('renders without crashing', function() {
  shallow(<Header />);
});

it('matches snapshot', function() {
  let wrapper = shallow(<Header />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
