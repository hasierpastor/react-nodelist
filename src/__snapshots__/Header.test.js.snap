// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<h1
  style={
    Object {
      "fontFamily": "raleway",
      "fontSize": "4rem",
      "margin": "1rem",
    }
  }
>
  Tree Table 
  <FontAwesomeIcon
    border={false}
    className=""
    fixedWidth={false}
    flip={null}
    icon="tree"
    inverse={false}
    listItem={false}
    mask={null}
    pull={null}
    pulse={false}
    rotation={null}
    size={null}
    spin={false}
    style={
      Object {
        "color": "#228B22",
      }
    }
    symbol={false}
    title=""
    transform={null}
  />
</h1>
`;
